doc/development-guidelines/conda/meta.yaml.template

```diff
@@ -27,15 +27,10 @@ requirements:
     host:
         - pip
         - python {{ python }}
-        - setuptools
-        - setuptools_scm
-    build:
-        - python {{ python }}
-        - setuptools
-        - setuptools_scm
+        - setuptools >={{ setuptools}}
```

Comment:
couldn't you add these to ts-conda-build and then add ts-conda-build here as a build dependency?